frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx

```diff
@@ -57,6 +58,11 @@ export const LogDetails = ({ row, queryTerms }: Props) => {
 	const expandable = Object.values(logAttributes).some(
 		(v) => typeof v === 'object',
 	)
+	const attributesShownWhenPresent = {
```

Comment:
@ccschmitz - per #4672, can adjust this to `ReservedLogKey`: https://github.com/highlight/highlight/blob/main/backend/private-graph/graph/schema.graphqls#L533-L541

Reply:
We might need to add `message` to that enum.

Reply:
with #4672, we'll want `Source` and `ServiceName` as part of this as well.